src/applications/claims-status/tests/components/claim-files-tab/RemoveFileModal.unit.spec.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import RemoveFileModal from '../../../components/claim-files-tab/RemoveFileModal';
+
+describe('<RemoveFileModal>', () => {
+  const props = {
+    removeFileName: 'test.pdf',
+    showRemoveFileModal: true,
```

Comment:
you can set closeModal and removeFile here as well so you dont have to set it in the tests. 
EX:
closeModal: () => {},
removeFile: () => {},